cddl_gen/cddl_gen.py

```diff
@@ -220,10 +220,15 @@ def strip_comments(instr):
         comment_regex = r"\;.*?\n"
         return sub(comment_regex, '', instr)
 
+    @staticmethod
+    def resolve_backslashes(instr):
+        return sub(r"\\\n", " ", instr)
```

Comment:
Won't this also delete the newline character? If so that is not what is described in the commit message.